sjb/config/test_cases/test_branch_origin_service_catalog_310.yml

```diff
@@ -1,6 +1,8 @@
 ---
 parent: 'common/test_cases/origin_minimal.yml'
 overrides:
+  sync:
+    - "openshift,origin=release-3.10"
```

Comment:
If you are picking up other syncs from the parent, this will override all of them -- ensure you are not missing any